app/Api/Controllers/Alerting/AlertsController.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/*
+ * Copyright (C) 2016 Neil Lathwood <neil@lathwood.co.uk>
```

Comment:
Your copyright/phpdoc header isn't matching the template here: https://github.com/librenms/librenmsv2/blob/develop/docs/Developing/File-Header.md